### PR TITLE
Add vendor into tidb-tools

### PR DESCRIPTION
**Pull request by shenli**
@siddontang @qiuyesuifeng @tiancaiamao PTAL

### CODE REVIEW

**Comment by siddontang** [comment]
Because tidb-tools is not a library, I think we can put vendor directly in root path, not _vendor.

**Comment by shenli** [comment]
@siddontang PTAL

File: Makefile [hunk 1]
```diff
@@ -1,14 +1,23 @@
-GO=GO15VENDOREXPERIMENT="1" go
+CURDIR := $(shell pwd)
+path_to_add := $(addsuffix /bin,$(subst :,/bin:,$(CURDIR)/_vendor:$(GOPATH)))
```
Comment (siddontang) [inline Makefile]: no need use $(CURDIR)/_vendor now

**Comment by tiancaiamao** [comment]
1. check Makefile, remove _vendor relate things, including PATH, GOBUILD, and so on
2. it seems many unnecessary files introduced to vendor, try
```
go get -v -u github.com/sgotti/glide-vc
glide vc --only-code --no-tests
```
see https://github.com/pingcap/tidb/blob/master/Makefile#L121 for example

**Comment by shenli** [comment]
@tiancaiamao PTAL

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**